frontend/src/axios.js

```diff
@@ -1,13 +1,14 @@
 import axios from "axios";
 
 export default function api(config) {
-  config = config === undefined && {};
+  config = config === undefined ? {} : config;
```

Comment:
your changed line does exactly as in the previous version of mine, just the longer version.

Reply:
I think so too, but for some reason it always fell to undefined when I ran it locally. So no tokens were sent in the header.

Reply:
yes you are right actually, i can see that happenning. config's value must be turning to "false" boolean. Your version is better